src/transformers/models/wav2vec2/feature_extraction_wav2vec2.py

```diff
@@ -172,14 +179,6 @@ def __call__(
             and (isinstance(raw_speech[0], np.ndarray) or isinstance(raw_speech[0], (tuple, list)))
         )
 
-        # make sure input is in list format
```

Comment:
Currently all the padding is happening in pure python and not in numpy so let's move the numpification further down